lib/tournament.json

```diff
@@ -625,6 +625,44 @@
       }
     ]
   ],
+  "rankings": [
```

Comment:
Fair enough.  I could do some work to massage the data into a consolidated format before I start rendering.